configs/eelbot/config.yaml

```diff
@@ -60,43 +60,12 @@ commands:
     enable: true
 
 replies:
-  # Reply to all caps messages.
-  caps:
-    enable: true
-    min_len: 5 # Minimum number of characters in message to consider match (ie. avoid short acronyms).
-    percent: 17 # Percent chance to trigger reply on match.
-    min_delay: 3 # Minimum time (in seconds) to wait before replying.
-    max_delay: 6 # Maximum time (in seconds) to wait before replying.
-    timeout: 120 # Timeout (in seconds) for consequetive replies.
-
-  # Reply to hello messages.
-  hello:
-    enable: true
-    percent: 33 # Percent chance to trigger reply on match.
-    min_delay: 3 # Minimum time (in seconds) to wait before replying.
-    max_delay: 6 # Maximum time (in seconds) to wait before replying.
-    timeout: 600 # Timeout (in seconds) for consequetive replies.
-
-  # Reply to goodbye messages.
-  goodbye:
-    enable: true
-    percent: 33 # Percent chance to trigger reply on match.
-    min_delay: 3 # Minimum time (in seconds) to wait before replying.
-    max_delay: 6 # Maximum time (in seconds) to wait before replying.
-    timeout: 600 # Timeout (in seconds) for consequetive replies.
-
-  # Reply to laugh messages.
-  laugh:
-    enable: true
-    percent: 17 # Percent chance to trigger reply on match.
-    min_delay: 1 # Minimum time (in seconds) to wait before replying.
-    max_delay: 3 # Maximum time (in seconds) to wait before replying.
-    timeout: 10 # Timeout (in seconds) for consequetive replies.
-
-  # Reply to messages that only contain the questionmark character.
-  question:
-    enable: true
-    percent: 17 # Percent chance to trigger reply on match.
-    min_delay: 3 # Minimum time (in seconds) to wait before replying.
-    max_delay: 6 # Maximum time (in seconds) to wait before replying.
-    timeout: 10 # Timeout (in seconds) for consequetive replies.
+  # Add custom replies by creating a table under the `reply` schema in the database first, then adding an entry here to
+  # reference it. Check the README for more details.
+  - table_name: "myreply"
+    regexps:
+    - "a{3,6}"
+    percent: 33
+    min_delay: 3
+    max_delay: 6
+    timeout: 600
```

Comment:
If we are going to do this, IMO this needs to live in the DB not in config, it doesn't fully achieve the goal of being able to extend replies to <many prompts> -> <many reply options> easily / with eelbot-ui this way.

Reply:
The reason I did this is because the config is loaded once at startup whereas the DB tables can be changed at runtime (replies added/removed etc.). I felt like putting the config in the DB could imply that this too could be changed on the fly. I'll move the config to the DB now, and in a future MR perhaps, I think hot reloading of the config can be handled in the app by watching for changes (not sure how easy it is in Postgres).

Reply:
> (not sure how easy it is in Postgres).

Not difficult IIUC. But I figure what we will more likely want to consider is having eelbot-ui signal to eelbot about changes rather than a DB level event.

> I felt like putting the config in the DB could imply that this too could be changed on the fly

Yep, suspected this was the reason. But I do think we should build this around the hopefully realistic future case where eelbot-ui is going to be used to enter in a lot of new reply types without having to re-deploy eelbot often.